foodalert/test/test_notification.py

```diff
@@ -94,9 +94,13 @@ def test_get_notification_list(self):
 
         self.assertEqual(len(response.json()), 2)
 
+        # reverse order of list
+        check = self.test_data[:2]
+        check.reverse()
```

Comment:
Should probably change the order of the test data so it matches the new reality of the API. Should remember to update the documentation too.